modules/createHistory.js

```diff
@@ -81,6 +81,9 @@ function createHistory(options={}) {
 
     return function () {
       changeListeners = changeListeners.filter(item => item !== listener)
+      if (changeListeners.length === 0) {
+        location = undefined
```

Comment:
Make this `null` instead of `undefined` and I'll merge.

Going to follow up w/@mjackson on getting a release out.